### ISSUE TITLE
Offline mobile version

### ISSUE THREAD

**Issue by marcheimendinger** [AvolitesLtd/TitanManual#80]
What about an iOS/Android app with an offline version of the manual?

**Comment by Farrser** [comment]
Yes an offline app version of the manual is very much something we are interested in. The current plan for this is we will get it in the format of a PWA for free when we update to [Docusaurus 2](https://v2.docusaurus.io/).

Currently Docusaurus 2 is still in alpha so and they are potentially still making breaking changes so we don't want to upgrade just yet. When it becomes a bit more stable and has proper [translation support](https://github.com/facebook/docusaurus/issues/2651) we will look at migrating over. We'll then get an offline app!

**Comment by garethnunns** [comment]
At that point a dedicated app could be made with [Expo](https://expo.io/)

**Comment by Farrser** [comment]
I suppose this could happen fairly easily with Expo regardless of the PWA support in a similar way to the desktop app works
